<!DOCTYPE HTML>
<html>

<head>
  <title>shadowplay_2 - another page</title>
  <meta name="description" content="website description" />
  <meta name="keywords" content="website keywords, website keywords" />
  <meta http-equiv="content-type" content="text/html; charset=windows-1252" />
  <link rel="stylesheet" type="text/css" href="style/style.css" />
</head>

<body>
  <div id="main">
    <div id="header">
      <div id="logo">
        <div id="logo_text">
          <!-- class="logo_colour", allows you to change the colour of the text -->
          <h1><a href="index.html">shadow<span class="logo_colour">play_2</span></a></h1>
          <h2>Simple. Contemporary. Website Template.</h2>
        </div>
      </div>
      <div id="menubar">
        <ul id="menu">
          <!-- put class="selected" in the li tag for the selected page - to highlight which page you're on -->
          <li><a href="index.html">Home</a></li>
          <li><a href="examples.html">Examples</a></li>
          <li><a href="page.html">A Page</a></li>
          <li class="selected"><a href="another_page.html">Another Page</a></li>
          <li><a href="contact.html">Contact Us</a></li>
        </ul>
      </div>
    </div>
    <div id="content_header"></div>
    <div id="site_content">
     
      <div id="content">
        <!-- insert the page content here -->
        <div id="References2" class="shiny-html-output shiny-bound-output"><br>
<h4>
  <b>User manuals and tutorials:</b>
</h4>
<ul>
  <li>
    <a href="https://www.bioconductor.org/packages/release/bioc/vignettes/Prostar/inst/doc/Prostar_UserManual.pdf?attredirects=0" target="_blank">ProStaR user manual</a>
  </li>
  <li>
    <a href="https://sites.google.com/site/thomasburgerswebpage/download/Prostar_Tutorial.pdf?attredirects=0" target="_blank">ProStaR Tutorial</a>
  </li>
  <li>
    <a href="https://sites.google.com/site/thomasburgerswebpage/download/tutorial-CP4P-4.pdf?attredirects=0" target="_blank">CP4P Tutorial</a>
  </li>
</ul>
<br>
<h4>
  <b>Reference manuals:</b>
</h4>
<ul>
  <li>
    <a href="https://www.bioconductor.org/packages/release/bioc/manuals/Prostar/man/Prostar.pdf" target="_blank">ProStaR reference manual</a>
  </li>
  <li>
    <a href="https://www.bioconductor.org/packages/release/bioc/manuals/DAPAR/man/DAPAR.pdf?attredirects=0" target="_blank">DAPAR reference manual</a>
  </li>
  <li>
    <a href="https://www.bioconductor.org/packages/release/bioc/html/MSnbase.html" target="_blank">MSnbase package webpage</a>
  </li>
  <li>
    <a href="https://cran.r-project.org/web/packages/cp4p/cp4p.pdf?attredirects=0" target="_blank">CP4P reference manual</a>
  </li>
  <li>
    <a href="https://cran.r-project.org/web/packages/imp4p/imp4p.pdf?attredirects=0" target="_blank">IMP4P reference manual</a>
  </li>
</ul>
<br>
<h4>
  <b>References:</b>
</h4>
<ol>
  <h4>
    <b>Our referenced works:</b>
  </h4>
  <li>
    <a href="https://sites.google.com/site/thomasburgerswebpage/download/calib-final.pdf?attredirects=0" target="_blank">
      <p>
        Q. Giai Gianetto, F. Combes, C. Ramus, C. Bruley, Y. Coute and 
  T. Burger. Calibration Plot for Proteomics (cp4p): A graphical tool 
                to visually check the assumptions underlying FDR control in 
                quantitative experiments.
        <i>Proteomics</i>
        , 16(1):29-32, 2016.
      </p>
    </a>
  </li>
  <li>
    <a href="https://sites.google.com/site/thomasburgerswebpage/download/natureOfMV-Vsubmited2.pdf?attredirects=0" target="_blank">
      <p>
        C. Lazar, L. Gatto, M. Ferro, C. Bruley, T. Burger. Accounting 
  for the multiple natures of missing values in label-free quantitative 
                proteomics datasets to compare imputation strategies.
        <i>Journal of 
                Proteome Research</i>
        , 15(4):1116-1125, 2016.
      </p>
    </a>
  </li>
  <li>
    <a href="https://sites.google.com/site/thomasburgerswebpage/download/OnTheMissuseOfFudgeFactorInProteomics_FV.pdf?attredirects=0" target="_blank">
      <p>
        Q. Giai Gianetto, Y. Coute, C. Bruley and T. Burger. Uses and 
  misuses of the fudge factor in quantitative discovery proteomics.
        <i>Proteomics</i>
        , 16(14):1955-60, 2016.
      </p>
    </a>
  </li>
  <li>
    <a href="https://sites.google.com/site/thomasburgerswebpage/download/prostar.pdf?attredirects=0" target="_blank">
      <p>
        S. Wieczorek, F. Combes, C. Lazar, Q. Giai-Gianetto, L. Gatto, 
  A. Dorffer, A.-M. Hesse, Y. Coute, M. Ferro, C. Bruley and T. Burger. 
                DAPAR &amp; ProStaR: software to perform statistical analyses in 
                quantitative discovery proteomics,
        <i>Bioinformatics</i>
        , 33(1):135-136, 2017
      </p>
    </a>
  </li>
  <li>
    <a href="https://sites.google.com/site/thomasburgerswebpage/download/fdrtuto.pdf?attredirects=0" target="_blank">
      <p>
        T. Burger. Gentle introduction to the statistical 
  foundations of false discovery rate in quantitative proteomics.
        <i>Journal of Proteome Research</i>
        , 17(1):12-22, 2017.
      </p>
    </a>
  </li>
  <li>
    <a href="https://sites.google.com/site/thomasburgerswebpage/download/revised-biostat-proteom-preprint.pdf?attredirects=0" target="_blank"> L. Jacob, F. Combes and T. Burger. PEPA test : fast and powerful differential analysis
    from relative quantitative proteomics data using shared peptides. (accepted for publication, 2018).</a>
  </li>
  <li>
    <p>Q. Giai Gianetto, C. Lazar, S. Wieczorek, C. Bruley, Y. Coute and 
    T. Burger. Multiple imputation strategy for mass spectrometry-based 
                proteomic data. (in preparation).</p>
  </li>
  <h4>
    <b>Other references:</b>
  </h4>
  <li>
    <p>Bolstad BM (2017). preprocessCore: A collection of pre-processing functions. R package version 1.38.1</p>
    <a href="https://github.com/bmbolstad/preprocessCore" target="_blank">link</a>
  </li>
  <li>
    <p>Hastie T, Tibshirani R, Narasimhan B and Chu G (2017). impute: Imputation for microarray data. R package version 1.50.1</p>
  </li>
  <li>
    <p>
      Gatto L and Lilley K (2012). 
    MSnbase - an R/Bioconductor package for isobaric tagged mass spectrometry data visualization, processing and quantitation.
      <i>Bioinformatics</i>
      , 28, pp. 288-289.
    </p>
  </li>
  <li>
    <p>
      Ritchie ME, Phipson B, Wu D, Hu Y, Law CW, Shi W and Smyth GK (2015). 
    Limma powers differential expression analyses for RNA-sequencing and microarray studies.
      <i>Nucleic Acids Research</i>
      , 43(7), pp. e47.
    </p>
  </li>
</ol></div>
      </div>
    </div>
    <div id="content_footer"></div>
    <div id="footer">
      <p><a href="index.html">Home</a> | <a href="examples.html">Examples</a> | <a href="page.html">A Page</a> | <a href="another_page.html">Another Page</a> | <a href="contact.html">Contact Us</a></p>
      <p>Copyright &copy; shadowplay_2 | <a href="http://validator.w3.org/check?uri=referer">HTML5</a> | <a href="http://jigsaw.w3.org/css-validator/check/referer">CSS</a> | <a href="http://www.html5webtemplates.co.uk">design from HTML5webtemplates.co.uk</a></p>
    </div>
  </div>
</body>
</html>
